src/views/preview/presentation.jsx

```diff
@@ -37,123 +53,208 @@ const PreviewPresentation = props => {
-                            <button className="button remix-button">
-                                Remix
-                            </button>
-                            <button className="button see-inside-button">
-                                See Inside
-                            </button>
-                        </div>
-                    </FlexRow>
-                    <FlexRow className="preview-row">
-                        <div className="placeholder">
-                            <img src={placeholder} />
-                        </div>
-                        <FlexRow className="project-notes">
-                            {shareDate && (
-                                <div className="share-date">
-                                    <div className="copyleft">&copy;</div>
-                                    {' '}
-                                    {/*  eslint-disable react/jsx-sort-props */}
-                                    <FormattedDate
-                                        value={Date.parse(shareDate)}
-                                        day="2-digit"
-                                        month="short"
-                                        year="numeric"
-                                    />
-                                    {/*  eslint-enable react/jsx-sort-props */}
+                    <Formsy>
+                        <FlexRow className="preview-row">
+                            <FlexRow className="project-header">
+                                <Avatar
+                                    src={`https://cdn2.scratch.mit.edu/get_image/user/${projectInfo. author.id}_48x48.png`}
+                                />
+                                <div className="title">
+                                    {editable ?
+                                        
+                                        <InplaceInput
+                                            className="project-title"
+                                            handleUpdate={onUpdate}
+                                            name="title"
+                                            validationErrors={{
+                                                maxLength: 'Sorry title is too long'
+                                                // maxLength: props.intl.formatMessage({
+                                                //     id: 'project.titleMaxLength'
+                                                // })
+                                            }}
+                                            validations={{
+                                                // TODO: actual 100
+                                                maxLength: 40
+                                            }}
+                                            value={projectInfo.title}
+                                        /> :
+                                        <div className="project-title">{projectInfo.title}</div>
```

Comment:
Eventually it might be nice to consolidate this kind of switching into a component that accepts an editable prop.